.github/workflows/tests-cleanup.yml

```diff
@@ -60,14 +60,19 @@ jobs:
         } 
     - name: Notify Slack on failure
       if: ${{ failure() }}
-      uses: 8398a7/action-slack@v3.12.0
-      with:
-        status: ${{ job.status }}
-        text: NServiceBus.AmazonSQS clean up has failed.
-        author_name: Amazon SQS Transport Test Cleanup
-        fields: message,author,action,eventName,ref,workflow
-        username: GitHub Actions
-        icon_emoji: ":github_actions:"
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        SLACK_WEBHOOK_URL: ${{ secrets.SLACK_AWS_CHANNEL_WEBHOOK }}
+      shell: pwsh
+      run: |
+          $headers = @{
+              'Authorization' = "Bearer ${{ secrets.GITHUB_TOKEN }}"
+          }
+          $body = @{
+            channel = 'aws'
+            text = "NServiceBus.AmazonSQS clean up has failed: https://github.com/Particular/NServiceBus.AmazonSQS/actions/workflows/tests-cleanup.yml"
+            username = 'Amazon SQS Transport Test Cleanup'
+            icon_emoji = 'github_actions'
+            unfurl_links = false
+            unfurl_media = false
+          } | ConvertTo-Json
+          $result = Invoke-RestMethod -Method POST -Uri https://slack.com/api/chat.postMessage -ContentType "application/json; charset=utf-8" -Headers $headers -Body $body
+          Write-Output $result
```

Comment:
Could we every leak tokens into the output here?

Reply:
Nope: https://github.com/Particular/docs.particular.net/pull/5608/files#r790814025